www/js/components/rating-point-card.js

```diff
@@ -0,0 +1,158 @@
+import React, {Component} from 'react';
+import { Card, CardActions, CardText,
+  RaisedButton, Paper, TextField } from 'material-ui';
+import RatingSelector from './rating-selector';
+import PointHeader from './point-header';
+
+export class RatingPointCard extends Component {
+
+  onSubmit() {
+    const { newRating, newComment } = this.props;
+    // TODO: submit this comment
+    console.log(`rating: ${newRating.rating}`);
+    console.log(`comment: ${newRating.comment}`);
+    console.log(`date: ${new Date().toJSON()}`)
+  }
+
+  onCommentUpdate(event) {
+    const {setComment} = this.props;
+    const comment = event.target.value;
+    setComment(comment);
+  }
+
+  render() {
+    const { peekMarker, deselectMarker, point,
+      setRating, newRating, setComment, history } = this.props;
+
+    const headerHeight = Math.max(64,  64 + this.props.heightOffset);
+    const headerHeightCSS = `calc(100% - ${headerHeight}px)`;
+    let cardStyle = {
+      width: '100%',
+      position: 'fixed',
+      bottom: '0px',
+      height: headerHeightCSS,
+      transition: 'all 300ms ease',
+      zIndex:'8',
+      overflowY:'auto'
+    }
+
+    let seeButton = (
+      <RaisedButton
+        secondary
+        label="See Less"
+        onClick={() => {
+          peekMarker();
+        }}
+      />
+    );
+
+    const mockComments = [
+      {
+        'user': "gypsy",
+        'rating': 0,
+        'text': "This place was a great... place.",
+        'date': "2016-02-28T15:37:04.540Z"
+      },
+      {
+        'user': "cambot",
+        'rating': 3,
+        'text': "",
+        'date': "2016-02-28T15:37:04.540Z"
+      },
+      {
+        'user': "tomservo",
+        'rating': 3,
+        'text': "This place was okay",
+        'date': "2016-02-28T15:37:04.540Z"
+      },
+      {
+        'user': "crow",
+        'rating': 5,
+        'text': "This place smells like pepperoni... I love pepperoni!",
+        'date': "2016-02-28T15:37:04.540Z"
+      }
+    ]
```

Comment:
This should really be in it's own component that you can import... but not critical for this PR